docs/layers/grid-layer.md

```diff
@@ -5,19 +5,21 @@
   <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="extruded" />
 </p>
 
-# GridLayer
+# Grid Layer
```

Comment:
I think we need to go with one style or another. Split all layer names or none. 

If this is reference documentation, it makes sense to keep the names without spaces to be identical to the actual layer names.

Under any circumstances, the text in the index on the left, and the title of each doc should be the same.